subsys/bluetooth/services/fast_pair/fp_gatt_service.c

```diff
@@ -95,8 +96,10 @@ struct msg_seekers_passkey {
 	uint32_t passkey;
 };
 
-/* Reference to the Fast Pair information callback structure. */
-static const struct bt_fast_pair_info_cb *fast_pair_info_cb;
+/* Reference to the Fast Pair information callback list. */
```

Comment:
`Fast Pair information callback list` (it's actually a list object, not reference to it)